.env

```diff
@@ -0,0 +1 @@
+JWT_SECRET=test
```

Comment:
I know your secret(s); with real apps, you'd definitely want your .env in your .gitignore

Reply:
Oh man! Thank you for reminding me!